tests/cache/hardhat-network-fork/network-1/request-0a323c2abcb5d30f127be1560fe14bc5.json

```diff
@@ -0,0 +1 @@
+["0x","0x330","0xadb360ce69138b26bb"]
```

Comment:
By including these cache files, we don't ever need to make requests to Alchemy.

Reply:
I think using these cached values will help with plugin testing and we can keep an eye toward creating the CI workflow of testing that can do more complex and 'live' calls as we need them.